romana-install/roles/romana/install-agent/tasks/files.yml

```diff
@@ -1,4 +1,10 @@
 ---
+- name: Ensure install path exists
+  file: path="{{ item }}" state=directory
+  with_items:
+    - "{{ romana_dir }}"
+    - "{{ romana_dir }}/bin"
```

Comment:
Great point. In a "proper" installation, where should romana_dir be?
I was tempted to put binaries in /usr/local/bin and configs in /etc/romana -- that's how some other things are set up. Having not decided, I didn't change the behaviour from current master.